background/storage.ts

```diff
@@ -111,6 +125,18 @@ export type PasskeyAccount = {
   address: HexString
 }
 
+export type imAccount = {
+  type: AccountType.imAccount
+  chainId: number
+  address: HexString
+  ecdsaValidator: HexString
+  ownerPrivateKey: HexString
+  webAuthnValidator: HexString
```

Comment:
future: Just a note that we may need a nested structure to model the validator in the future, like the account, for instance:

```ts
type imAccount = {
    // ...
    ownerValidator: {
        type: imAccountValidatorType.ECDSA,
        ownerPrivateKey: 0x1234,
        // ... other fields
    }
}
```